packages/wdio-allure-reporter/tests/cucumber.suite.test.ts

```diff
@@ -393,22 +404,29 @@ describe('reporter option "useCucumberStepReporter" set to true', () => {
             reporter.onAfterCommand(commandEnd())
             reporter._consoleOutput = 'some console output'
             reporter.onTestFail(cucumberHelper.testFail())
-            const suiteResults: any = { tests: ['failed'] }
+
+            const suiteResults: any = { tests: [{ state: 'failed' }] }
+
             reporter.onSuiteEnd(cucumberHelper.scenarioEnd(suiteResults))
             reporter.onSuiteEnd(cucumberHelper.featureEnd(suiteResults))
             reporter.onRunnerEnd(runnerEnd())
-            const results = getResults(outputDir)
-            expect(results).toHaveLength(1)
-            allureXml = results[0]
 
-            expect(allureXml('ns2\\:test-suite > name').text()).toEqual('MyFeature')
-            expect(allureXml('test-case > name').text()).toEqual('MyScenario')
-            expect(allureXml('test-case').attr('status')).toEqual('failed')
-            expect(allureXml('step').attr('status')).toEqual('failed')
-            expect(allureXml('test-case parameter[kind="argument"]')).toHaveLength(1)
-            expect(allureXml('test-case parameter[name="browser"]').eq(0).attr('value')).toEqual('chrome-68')
-            expect(allureXml('test-case > steps > step > attachments > attachment')).toHaveLength(1)
-            expect(allureXml('test-case > steps > step > attachments > attachment').eq(0).attr('title')).toBe('Console Logs')
+            const { results, containers } = getResults(outputDir)
+
+            allureResult = results[0]
+            allureContainer = containers[0]
+
+            const browserParameter = allureResult.parameters.find((param: Parameter) => param.name === 'browser')
+
+            expect(results).toHaveLength(1)
```

Comment:
please assert length before you get value

Reply:
:+1: